clients/java-logger/library/src/main/java/com/abcxyz/lumberjack/auditlogclient/AuditLoggingServerInterceptor.java

```diff
@@ -313,14 +315,17 @@ Optional<Selector> getRelevantSelector(String methodIdentifier) {
 
   /**
    * Get the justification token out of the headers, and create a struct of the correct format for
-   * use in audit logging metadata. We expect the claims from the JWT to be put in JSON format and
-   * put into the struct under the justification_token key.
+   * use in audit logging context.
    */
-  AuditLog.Builder setJustification(Metadata headers, AuditLog.Builder logBuilder)
-      throws LogProcessingException {
-    Metadata.Key<String> metadataKey =
-        Metadata.Key.of("justification-token", Metadata.ASCII_STRING_MARSHALLER);
-    String jvsToken = headers.get(metadataKey);
-    return justificationProcessor.auditLogBuilderWithJustification(jvsToken, logBuilder);
+  Struct getAuditLogRequestContext(Metadata headers) {
+    String jvsToken = headers.get(JUSTIFICATION_METADATA_KEY);
+    if (jvsToken != null && jvsToken.length() > 0) {
```

Comment:
For checking if string is not empty we should use `!jvsToken.isEmpty()` or `!jvsToken.isBlank()` rather than the length check.